src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,117 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  query: string;
+  sortBy: string;
+  selectedTodo?: Todo | null;
```

Comment:
there is no need to add mark `?` because you wrote a default value ` selectedTodo = null,`
so anyway it will be null if we don't pass any todo